stable/database/files/nuosm

```diff
@@ -278,11 +280,22 @@ function perform_restore() {
       rm -rf $download_dir
     fi
 
+    if [ "$SEPARATE_JOURNAL" == "true" ] && [ "$restore_type" == "stream" ]; then
+      # `nuodocker restore archive` doesn't invoke `nuoarchive` on cold archive
+      # (stream) restore type. Move the content of the internal journal
+      # directory to the external journal path manually as the journal is not
+      # consumed during SM shutdown
+      log "Moving restored journal content to $JOURNAL_DIR"
+      mv "$download_dir/journal/"* "$JOURNAL_DIR"
```

Comment:
There is a risk that, given someone using an environment with a separate archive chooses to prep a _stream_ type of restore, that they are manually moving the journal back into the archive and may not put it inside /journal/.  We have example of this with some customers.  It's probably too much to allow the journal source location within the tar to be configurable, but maybe there is a good place to comment in the restore docs.

Reply:
Good point. I will add a note in the B&R document about this rather than making it configurable.